lib/tasks/play.js

```diff
@@ -66,8 +67,20 @@ class TaskPlay extends Task {
         }
       }
     } catch (err) {
-      if (timeout) clearTimeout(timeout);
       this.logger.info(err, `TaskPlay:exec - error playing ${this.url}`);
```

Comment:
instead of logging the entire err object, can we log the err.message?

Reply:
this would be a good idea, stack traces are not helpful in this case and can confuse someone reading the logs @sammachin